dvc/repo/get.py

```diff
@@ -17,6 +17,9 @@
 def get(url, path, out=None, rev=None):
     out = out or os.path.basename(urlparse(path).path)
 
+    if os.path.splitext(out)[1] == ".dvc":
```

Comment:
do we have a helper somewhere that detects that it's a DVC file

Reply:
yes, @shcheklein :

```python
# stage.py
    @staticmethod
    def is_stage_file(path):
        return os.path.isfile(path) and Stage.is_valid_filename(path)
```

I'll use that one instead.